libraries/AC_AttitudeControl/LogStructure.h

```diff
@@ -92,6 +93,28 @@ struct PACKED log_Rate {
     float   throttle_slew;
 };
 
+// @LoggerMessage: ANG
+// @Description: Canonical vehicle attitude
+// @Field: TimeUS: Time of gyro sample
```

Comment:
Maybe "attitude sample" its a bit odd mentioning gyro here on a angle message.

Reply:
Maybe: "Time stamp of the current Attitude loop"

Reply:
Fixed